### ISSUE TITLE
Don't open on new tab

### ISSUE THREAD

**Issue by AckslD** [kyazdani42/nvim-tree.lua#373]
Whenever I open a new tab, nvim-tree also opens, even if it wasn't open before. This behaviour seems to have changed recently and if I understand the docs it shouldn't be the behaviour. My settings are:
```
vim.g.nvim_tree_auto_open = 0
vim.g.nvim_tree_auto_ignore_ft = {'startify'}
vim.g.nvim_tree_auto_close = 1
vim.g.nvim_tree_quit_on_open = 1
vim.g.nvim_tree_follow = 1
vim.g.nvim_tree_indent_markers = 1
vim.g.nvim_tree_git_hl = 1
vim.g.nvim_tree_tab_open = 1
vim.g.nvim_tree_width_allow_resize  = 1
vim.g.nvim_tree_disable_netrw = 0
vim.g.nvim_tree_icons = {
    default = '',
    symlink = '',
    git = {
        unstaged = "✗",
        staged = "✓",
        unmerged = "",
        renamed = "➜",
        untracked = "★",
    },
    folder = {
        default = "",
        open = "",
        empty = "",
        empty_open = "",
        symlink = "",
    }
}
```
Setting `vim.g.nvim_tree_tab_open` to `0` solves the issue but then I loose the behaviour of keeping the tree open in new tabs if it already was open.

**Comment by kyazdani42** [comment]
fixed in 825cf570bfe

**Comment by AckslD** [comment]
Thanks @kyazdani42 I can confirm that it is working (a bit late :))
